### ISSUE TITLE
Unreported Pokedex Data Not Saving Properly

### ISSUE THREAD

**Issue by mwoerner77** [kwsch/PKHeX#3392]
**Describe the bug**
In the Pokedex tab, altering task progress but not marking them as reported does not save.

**To Reproduce**
1. Go-to Pokedex tab
2. Alter the data of any research task
3. Hit save
4. Export Save
5. No changes made in game (bug)

Also if you go back into the Pokedex tab after step 3 you'll also see none of your edits saved.

**Expected behavior**
Such data should be marked as unreported tasks that need to be taken to the professor

**Comment by ReignOfComputer** [comment]
Not sure if related to https://github.com/kwsch/PKHeX/issues/3386. Are you running on Release or latest commit (dev build)?

**Comment by drewlobel** [comment]
I am having the same issue, and I am using commit

**Comment by sora10pls** [comment]
Duplicate of #3386 and #3394 -- works fine on latest co
![Screenshot 2022-02-05 10-27-21](https://user-images.githubusercontent.com/17801814/152647969-4e44c242-2510-4d7c-a940-68fc8c1f39d6.png)
mmit.
